src/Components/Admins/AdminForm/AdminForm.jsx

```diff
@@ -68,58 +74,51 @@ const AdminForm = () => {
   };
 
   useEffect(async () => {
-    if (id) {
-      setAdminId(id);
-      const response = await fetch(`${process.env.REACT_APP_API_URL}/admins/${id}`, {
-        method: 'GET'
-      });
-      const data = await response.json();
-      setEdit(true);
-      setName(data.data.name);
-      setLastName(data.data.lastName);
-      setEmail(data.data.email);
-      setPassword(data.data.password);
-    } else {
-      setEdit(false);
+    try {
+      if (id) {
+        setEdit(true);
+        setName(admins.find((item) => item._id === id).name);
```

Comment:
you are doing always the same find. We should save the admin in a new variable and reuse it.
```suggestion
        const currentAdmin = admins.find(item => item._id === id);
        setName(currentAdmin);
```

Reply:
done!